test/e2e/adminapi_resources.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("[Admin API] List Azure resources action", func() {
 		resourceID := resourceIDFromEnv()
 
 		By("getting the resource group where cluster resources live in")
-		oc, err := clients.OpenshiftClusters.Get(ctx, _env.ResourceGroup(), clusterName)
+		oc, err := clients.OpenshiftClustersv20200430.Get(ctx, _env.ResourceGroup(), clusterName)
```

Comment:
For now E2E are using old API. So this basically scatters our e2e. Cluster will be created with new API. So any new future tests should be not API binded. In example.
If we create FIPs cluster, we should write tests which currently run in dev only and uses Kubernetes API to check it. Not ARO... 

:hankey: a bit. Not sure how we can do this without copy-paste test cases.

Reply:
I don't agree with this or see the problem.  Let's discuss tomorrow.